packages/jest-circus/src/legacy_code_todo_rewrite/jest-adapter-init.js

```diff
@@ -43,6 +43,32 @@ const initialize = ({
   global.fit = global.it.only;
   global.fdescribe = global.describe.only;
 
+  global.test.concurrent = (
```

Comment:
i'm not including `test.concurrent` into `jest-circus` as a core functionality. In the current state it's never going to work properly (can't use certain matchers/hooks/proper .skip/.only/etc).
I also think we shouldn't document it either. It will be harder to reimplement it properly in the future if people start using it